### PR TITLE
1.9.0 release updates

### PR DESCRIPTION
**Pull request by ctubbsii**

### CODE REVIEW

**Comment by keith-turner** [comment]
Seeing this error when building locally.

```
 ** ERROR: directory is already being watched! **
```

File: _posts/release/2018-04-23-accumulo-1.9.0.md [hunk 1]
```diff
@@ -1,39 +1,43 @@
...
-better support Hadoop 3 in the future. New methods were added to replace these methods which cause
-this release to be a 1.9.0 release.  These changes allow removal of commons config from Accumulo's 
-API in 2.0.0.  If using ClientConfiguration, then switching from existing constructors to the new
-static methods [create()], [fromFile()], or [fromMap()] will ensure your code works in 2.0.0.
+In [ACCUMULO-4611], public API in ClientConfiguration using commons config
+types was deprecated to better support Hadoop 3 in the future. New methods were
+added to replace these methods which cause this release to be a 1.9.0 release.
+These changes allow removal of commons config from Accumulo's API in 2.0.0.  If
+using ClientConfiguration, then switching from existing constructors to the new
+static methods [create()], [fromFile()], or [fromMap()] will ensure your code
```
Comment (keith-turner) [inline _posts/release/2018-04-23-accumulo-1.9.0.md]: I wanted to ensure these links worked correctly, so I built these changes locally.  The links worked.  When I clicked on them I saw the following in the terminal running jekyll

```
[2018-04-23 15:28:22] ERROR `/1.9/apidocs/resources/fonts/dejavu.css' not found.
```

File: _posts/release/2018-04-23-accumulo-1.9.0.md [hunk 2]
```diff
@@ -1,39 +1,43 @@
...
-Apache Accumulo 1.9.0 is a minor release on the 1.x branch. This release would be considered a
-maintainence release on 1.8 branch except there are some API additions which resulted in a new minor
-release. Users of 1.8.x versions of Accumulo should upgrade to 1.9.0.  There will be no more bug fix
-releases on the 1.8 branch. This release contains changes from 96 issues.  See [GitHub] and [JIRA]
-for a list of changes.
+Apache Accumulo 1.9.0 is a minor release on the 1.x branch. This release would
+be considered a maintainence release on 1.8 branch except there are some API
+additions which resulted in a new minor release. Users of 1.8.x versions of
+Accumulo should upgrade to 1.9.0. There will be no more bug fix releases on the
+1.8 branch. This release contains changes for nearly a hundred issues. See
```
[the remark this answers is not in the document]
Comment (ctubbsii) [reply _posts/release/2018-04-23-accumulo-1.9.0.md]: I made that change for you. :smiley_cat:

File: _posts/release/2018-04-23-accumulo-1.9.0.md (hunk 1 above)
Comment (ctubbsii) [reply _posts/release/2018-04-23-accumulo-1.9.0.md]: Shouldn't be a problem, but it is fixed in newer Java https://bugs.openjdk.java.net/browse/JDK-8093199

**Pull request merged**